prody/proteins/interactions.py

```diff
@@ -2014,17 +2014,23 @@ def calcDistribution(interactions, residue1, residue2=None, **kwargs):
                 LOGGER.info(i)
 
 
-def listLigandInteractions(PLIP_output):
+def listLigandInteractions(PLIP_output, **kwargs):
     """Create a list of interactions from PLIP output created using calcLigandInteractions().
     Results can be displayed in VMD. 
     
     :arg PLIP_output: Results from PLIP for protein-ligand interactions.
-    :type PLIP_output: PLIP object obtained from calcLigandInteractions() 
+    :type PLIP_output: PLIP object obtained from calcLigandInteractions()
+    
+    :arg output: parameter to print the interactions on the screen
+                 while analyzing the structure
+    :type output: 'info'     
```

Comment:
at the moment, the type here is str and 'info' is an example value that would go in the section above.

I'd suggest making this default *False* and have the user enter *True* if they want it instead.

Reply:
Done